skyline-vscode/src/skyline_session.ts

```diff
@@ -104,6 +100,8 @@ export class SkylineSession {
         console.log("Sending initialization request");
         const message = new pb.InitializeRequest();
         message.setProtocolVersion(5);
```

Comment:
5 seems like a magic number here

Reply:
It's the current version of the protobuf messages. Perhaps we need to save this in a macro (or the Typescript equivalent)